milestone2.ts

```diff
@@ -83,32 +83,35 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
-
+let findAverage: GetNumber = (data) =>{
+  let sum = 0;
+  data.forEach(x => sum += x.age);
+  return sum/data.length;
+};
 //console.log(findAverage(userData));
 
 // Question 2: Data Handling
 /* Use data handling function(s) to find the first index of someone from San Francisco (SF).
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data) => data.findIndex(x => x.city === "SF"); // Code here
 
 //console.log(findIndexAns(userData));
 
 // Question 3: Filtering data
 /* Use data handling function(s) to find all of the volunteers from California (CA) over an age threshold n
  */
 type GetVolunteers = (data: Volunteer[], minAge: number) => Volunteer[];
-let findCAOverN: GetVolunteers; // Code here
+let findCAOverN: GetVolunteers = (data, minAge) => data.filter(x => ((x.age > minAge) && (x.state === "CA"))); // Code here
```

Comment:
The extra parenthesis around each of the conditions in your filter function aren't necessary and can be removed for increased clarity